Tools/peg_generator/pegen/c_generator.py

```diff
@@ -265,20 +288,11 @@ def generate(self, filename: str) -> None:
             mode = int(self.rules["start"].type == "mod_ty")
             if mode == 1 and self.grammar.metas.get("bytecode"):
                 mode += 1
-        modulename = self.grammar.metas.get("modulename", "peg_parser")
+        modulename = self.grammar.metas.get("modulename", "parse")
```

Comment:
Hmm... Probably c_generator.py should not be generating a module at all.

Reply:
You mean that we should be generating the module with trailer meta of Tools/peg_generator/data/python.gram? If so, that seems better. What should EXTENSION_SUFFIX be in that case?

Reply:
Ideally even the Tools/peg_generator/ version doesn't generate a module, but a single public function that calls the parser (much like the version under Parser/), and there's a separate small handcrafted file that defines the module. Could be a followup PR.